<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_inflacion</class>
 <widget class="QWidget" name="form_inflacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbx_anio">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Enero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>febrero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>marzo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>abril</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>mayo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>junio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>julio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>agosto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>septiembre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>octubre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>noviembre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>diciembre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_enero">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_febrero">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_marzo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_abril">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_mayo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_junio">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_julio">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_agosto">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_septiembre">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_octubre">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_noviembre">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_diciembre">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guardar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_actualizar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buscar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
